test/new-e2e/pkg/runner/parameters/const.go

```diff
@@ -19,6 +19,8 @@ const (
 	ExtraResourcesTags StoreKey = "extra_resources_tags"
 	// KeyPairName config file parameter name
 	KeyPairName StoreKey = "key_pair_name"
+	// PrivateKeyPassphrase config file parameter name
+	PrivateKeyPassphrase StoreKey = "private_key_passphrase"
```

Comment:
To be consistent with existing test-infra naming.

```suggestion
	PrivateKeyPassword StoreKey = "private_key_password"
```